web/app/ui/metalakes/CreateCatalogDialog.js

```diff
@@ -36,6 +36,9 @@ import { yupResolver } from '@hookform/resolvers/yup'
 
 import { providers } from '@/lib/utils/initial'
 
+const nameRegex = /^[a-zA-Z_][a-zA-Z0-9_]*$/
+const keyRegex = /^[a-zA-Z_][a-zA-Z0-9-_.]*$/
```

Comment:
Can we defines these two variables in a common places so that it can be used by two files.

Reply:
DONE